core/editor.js

```diff
@@ -1710,6 +1710,10 @@
 					} );
 				}
 			}, interval );
+
+			if ( CKEDITOR.config.delayIfDetached_cancelInterval ) {
+				CKEDITOR.config.delayIfDetached_cancelInterval( clearInterval( intervalId ) );
+			}
```

Comment:
I believe that we should provide a wrapper over `clearInterval` method. Currently, an interval is cleared on the first code execution. This logic should be also moved inside `setInterval` callback, not outside.